src/components/ManagerPage/ManagerApp.js

```diff
@@ -2,40 +2,47 @@
 
 import React from 'react'
 import styled from 'styled-components'
-import { getIconByCoinType } from '@ledgerhq/currencies/react'
-
-import type { Currency } from '@ledgerhq/currencies'
 
 import Box, { Tabbable } from 'components/base/Box'
 import Text from 'components/base/Text'
 
 const Container = styled(Box).attrs({
   align: 'center',
   justify: 'center',
-  m: 1,
+  m: 2,
 })`
   width: 150px;
   height: 150px;
-  background: rgba(0, 0, 0, 0.05);
+  background: white;
 `
 
+// https://api.ledgerwallet.com/update/assets/icons/bitcoin
+
 const ActionBtn = styled(Tabbable).attrs({
   fontSize: 3,
 })``
 
+const AppIcon = styled.img`
+  display: block;
+  width: 50px;
+  height: 50px;
+`
+
 type Props = {
-  currency: Currency,
+  name: string,
+  icon: string,
   onInstall: Function,
   onUninstall: Function,
 }
 
 export default function ManagerApp(props: Props) {
-  const { currency, onInstall, onUninstall } = props
-  const Icon = getIconByCoinType(currency.coinType)
+  const { name, icon, onInstall, onUninstall } = props
+  const iconUrl = `https://api.ledgerwallet.com/update/assets/icons/${icon}`
   return (
     <Container flow={3}>
-      {Icon && <Icon size={24} />}
-      <Text>{currency.name}</Text>
+      <AppIcon src={iconUrl} />
```

Comment:
any reason why not making AppIcon accept the icon and move the src inside AppIcon comopnent?

Reply:
As `AppIcon` is only a styled.img, it forwards all its props to `img` tag.
If needed, yep we could change it to stateless component which will take icon as prop, but as the design is just a POC for now, I think we can keep it like that.